app/renderer/components/config_menu.js

```diff
@@ -43,6 +46,8 @@ module.exports = {
                     <a class="waves-effect waves-light btn" v-on:click="importTasks">Import Tasks</a>
                     <label><em class="warning-text">ALERT! this will override your previous tasks</em></label>
                 </div>
+
+                <shortcuts-learn />
```

Comment:
Is autoclosing tag supported in vue?
Make sure it is (I though you need to use open-closing tags)

Reply:
really autoclosing is bad
[Self closing tag](https://vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended)